test/salesforce/salesforce.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
```

Comment:
can you name this file read-write.go or something like that to make it more clear it's the test file for read and write?